oe_theme.theme

```diff
@@ -418,6 +420,16 @@ function oe_theme_preprocess_pattern_dialog(array &$variables): void {
   $variables['hidden'] = $variables['hidden'] ? 'true' : 'false';
 }
 
+/**
+ * Implements hook_preprocess_HOOK() for pattern gallery template.
+ */
+function oe_theme_preprocess_pattern_gallery__preview(array &$variables): void {
+  foreach ($variables['items'] as &$item) {
+    $item['thumbnail'] = ImageValueObject::fromArray($item['thumbnail']);
+    $item = GalleryItemValueObject::fromArray($item);
```

Comment:
Can't the  `$item['thumbnail'] = ImageValueObject::fromArray($item['thumbnail'])` be called within `GalleryItemValueObject::fromArray($item);`? So that we encapsulate the logic in the factory and users will only need to use `GalleryItemValueObject::fromArray($item)`.

Reply:
:+1: